xDrip/Model/Glucose/GlucoseReading.swift

```diff
@@ -86,18 +86,18 @@ final class GlucoseReading: Object {
         return allMaster.filter { $0.date >? sensorStartDate }
     }
     
-    static func lastMasterReadings(_ amount: Int) -> [GlucoseReading] {
-        return lastMaster(amount: amount, filter: { $0.calculatedValue !~ 0 && $0.rawValue !~ 0 })
+    static func lastReadings(_ amount: Int, forMaster: Bool) -> [GlucoseReading] {
```

Comment:
`master` and `slave` modes are enum cases, so let's replace `forMaster: Bool` with `for mode: DeviceMode`